core/src/main/resources/io/questdb/site/conf/log.conf

```diff
@@ -14,6 +14,10 @@ writers=file,stdout,http.min
 #rollEvery accepts: day, hour, minute, month
 #w.file.rollEvery=day  
 #w.file.rollSize=1g
+#lifeDuration accepts: a number followed by s, m, h, d, w, M, y for seconds, minutes, hours, etc.  
+#w.file.lifeDuration=1d
+#sizeLimit is the max fileSize of the log directory. Follows same format as rollSize
+#w.file.sizeLimit=1m
```

Comment:
The commented value should be larger than the commented `w.file.rollSize` which is `1g`. Would be also nice to add some validation in the log writer to enforce that.